src/middlewared/middlewared/plugins/system.py

```diff
@@ -1470,6 +1471,7 @@ async def do_update(self, data):
 
         return await self.config()
 
+    @rest_api_metadata(explicit_methods=['get'])
```

Comment:
Can we allow post, too?

Also maybe it should be uppercase

Reply:
We dont want in U4 for it to break again ;)

Reply:
`post` is already allowed ? This is going to add other `explicit` methods apart from the default one which is going to be configured automatically based on the endpoint's `accepts` like other endpoints.

Reply:
In this case the default one would be `post` and `get` is explicitly added. I can make it upper case :)